test/integration/testImage/yaml/generate.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# Running this file executes `sonobuoy gen plugin` numerous times
+# to reproduce all the YAML for the plugin definitions. It assumes
+# you have the yq tool and sonobuoy in your PATH.
+
+# Should generate/test more cases; this is just a starting point.
+
+sonobuoy gen plugin \
+--name=job-junit-passing-singlefile \
+--image=sonobuoy/testimage:v0.1 \
+--cmd="testImage" \
+--arg="single-file" \
+--arg="/resources/junit-passing-tests.xml" \
+--format="junit" > job-junit-passing-singlefile.yaml
+
+sonobuoy gen plugin \
+--name=job-raw-passing-singlefile \
+--image=sonobuoy/testimage:v0.1 \
+--cmd="testImage" \
+--arg="single-file" \
+--arg="/resources/hello-world.txt" \
+--format="raw" > job-raw-singlefile.yaml
+
+sonobuoy gen plugin \
+--name=ds-junit-passing-tar \
+--image=sonobuoy/testimage:v0.1 \
+--cmd="testImage" \
+--arg="tar-file" \
+--arg="/resources/hello-world.txt" \
+--arg="/resources/junit-multi-suite-single-failure.xml" \
+--format="junit" > ds-junit-passing-tar.yaml
```

Comment:
Including `ds` in the name here and below suggest that these should be DaemonSet plugins but they are Job plugins.